app/deployments/tests/test_api.py

```diff
@@ -2,6 +2,7 @@
 
 import geojson
 from rest_framework.test import APITestCase
+import pytest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pytest'](https://app.codacy.com/manual/gmri/buoy_barn/pullRequest?prid=4987082)